pymess/utils/logged_requests.py

```diff
@@ -1,9 +1,26 @@
-def generate_session(slug=None, related_objects=None):
+class DefaultTimeoutSessionMixin:
```

Comment:
I don't understand why requests doesn't have this settings by default. Timeout is often forgotten